IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/Types/IfcRailwayPartTypeEnum/Documentation.md

```diff
@@ -1 +1,21 @@
 The IfcRailwayPartTypeEnum defines the range of different types of railway part that can be specified.
+
+The Spatial structure of a Railway is sub-divided into several elements for lateral and vertical decomposition:
+
+* TRACKSTRUCTURE is for the Track (vertical decomposition)
+* SUPERSTRUCTURE is for the upper part of the railway (vertical decomposition)
+* LINESIDESTRUCTURE is for the lateral part of the railway  (lateral decomposition)
+
+!["Railway vertical and lateral spatial decomposition "](../../../../../../figures/IfcRailwayPartTypeEnum-global.png "Figure 1 &mdash; Railway decomposition")
+
+Railway track structure have longitudinal decomposition based on track specificities:
+
+* PLAINTRACKSUPESTRUCTURE for the plain line tracks (IfcElementAssemblyTypeEnum.TRACKPANEL).
+* DILATATIONSUPERSTRUCTURE for the area of Dilatation panels (IfcElementAssemblyTypeEnum.DILATATIONPANEL).
+* TURNOUTSUPERSTRUCTURE for the area of Turnouts (IfcElementAssemblyTypeEnum.TURNOUTPANEL).
+* TRACKSTRUCTUREPART for generic longitudinal decomposition if needed.
```

Comment:
...and for vertical. Again, according to the famous picture

Reply:
@evandroAlfieri,

Again in the table shared with infra this lineside sub structure is flagged to be a longitudinal decomposition.